uninstall.sh

```diff
@@ -42,14 +42,19 @@ opt_skip_cache_and_logs=""
 failed=false
 
 un=$(uname)
+unm=$(uname -m)
```

Comment:
```suggestion
```

Reply:
Added New commit to the pr resolving above suggestions. Thank you.